schema/provisioner.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+package rusk;
+option go_package = "github.com/dusk-network/rusk-schema;rusk";
+
+import "stake.proto";
+
+message Provisioner {
+    bytes public_key_bls = 1;
```

Comment:
Don't we agree that we'll keep only raw (uncompressed) pk in Stake Contract?

Reply:
I think the agreement was to keep it unchanged for now:
https://github.com/dusk-network/rusk-schema/blob/104c43114fbe083b2c2b110aed925702cadb71be/rusk.proto#L35-L39

Reply:
In the future, after we take a final decision regarding C-GO, we might change that

Reply:
If so, we need to add raw pk in Stake Contract impl. Do you agree with that?

Reply:
I'm not sure I understand what would be the change. We already have `bytes raw_public_key_bls = 2;` in the line below. That way, we have both keys available.

Reply:
One can be transformed into the other trivially in the rust layer. So this wouldn't prove a challenge to be provided via Rusk service

Reply:
One can be transformed into the other trivially in the golang layer as well. In both cases we've got costly overhead due to conversion.

Reply:
AFAIK the expensive cost of the node side is the C-GO call. This is not needed from the Rusk perspective, so I can just remove it from the protobuf - its meant to help the go layer.

As we discussed in the call, we were to keep things as they are, but I can also update this here. What is your proposed change? From Rusk perspective, both raw/compressed forms are trivial.